src/webgpu/api/validation/capability_checks/limits/maxTextureDimension1D.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { addLimitTestParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxTextureDimension1D';
+const { g, description } = makeLimitTestGroup(limit);
+export { g, description };
+
+g.test('at_over')
+  .desc(`Test using at and over ${limit} limit`)
+  .params(u => addLimitTestParams(u))
+  .fn(async t => {
+    const { limitTest, testValueName } = t.params;
+    await t.testDeviceWithRequestedLimits(
+      limitTest,
+      testValueName,
+      async ({ device, testValue, shouldError }) => {
+        await t.testForValidationErrorWithPossibleOutOfMemoryError(() => {
+          const texture = device.createTexture({
+            size: [testValue, 1, 1],
+            format: 'rgba8unorm',
+            dimension: '1d',
+            usage: GPUTextureUsage.TEXTURE_BINDING,
+          });
+
+          // this 'if' is only here because of a bug in Chrome
+          // that generates an error calling destroy on an invalid texture
```

Comment:
I'd add something like "but it doesn't affect the coverage of the test so it's okay to leave here"

Reply:
I don't think it is okay to leave it there but when I tried to add a `TODO` it old me I wasn't allowed to add `TODO`.  Code that's workaround around Chrome bugs doesn't belong in the CTS.

Reply:
I updated the comment